package.json

```diff
@@ -6,7 +6,7 @@
   "author": "The Gaia Team <dev-gaia@lists.mozilla.org>",
   "engineStrict": "true",
   "engines": {
-    "node": ">=0.10",
+    "node": ">=4.0",
```

Comment:
My suggestion is to make this `^4.0.0` and change the version number of this package to `4.0.0`.